src/trpc/router/restRouter.ts

```diff
@@ -16,9 +16,13 @@ import { getOrganizationUnitListSchema } from '../../modules/organizationUnitSch
 import { organizationUnitMethods } from '../../modules/organizationUnitMethods';
 import { getServiceListSchema } from '../../modules/serviceSchemas';
 import { serviceMethods } from '../../modules/serviceMethods';
+import { trimAndJoin } from '../../utils/trimAndJoin';
+import { config } from '../../config';
 
 import { tr } from './router.i18n';
 
+const getCorporateEmail = (login: string | null) => `${login}${config.corporateEmailDomain}`;
```

Comment:
what if there is no corporateEmailDomain in env?

Reply:
will send only login

Reply:
loginundefined 🙂